distribution/sdk/sample-twitter-source/src/main/java/org/codice/ddf/catalog/twitter/source/TwitterSource.java

```diff
@@ -183,10 +186,9 @@ private Metacard getMetacard(Status status) {
         metacard.setSourceId(id);
         metacard.setId(String.valueOf(status.getId()));
         metacard.setTitle(status.getText());
-        metacard.setMetadata("<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>" +
-                "<Resource>" +
-                "<name>" + status.getText() + "</name>" +
-                "</Resource>");
+        metacard.setMetadata(
+                "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>" + "<Resource>"
```

Comment:
✏️ I would consider making a static final const format string, or even moving it to a resource. The string would look like
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?><Resource><name>%s</name></Resource>
```
and you'd set it here with `metacard.setMetadata(String.format(formatString, status.getText())`

I suggest a resource only so you can avoid all of the escapes for quotation marks.